interfaces/acados_template/acados_template/c_templates_tera/acados_sim_solver.in.c

```diff
@@ -417,25 +417,27 @@ int {{ model.name }}_acados_sim_solve({{ model.name }}_sim_solver_capsule *capsu
 }
 
 
-void {{ model.name }}_acados_sim_batch_solve({{ model.name }}_sim_solver_capsule ** capsules, int N_batch)
-{
-{% if solver_options.num_threads_in_batch_solve > 1 %}
-    int num_threads_bkp = omp_get_num_threads();
-    omp_set_num_threads({{ solver_options.num_threads_in_batch_solve }});
-
-    #pragma omp parallel for
-{%- endif %}
-    for (int i = 0; i < N_batch; i++)
+{% if solver_options.with_batch_functionality %}
```

Comment:
In this templated logic, we don't want to indent, such that the rendered code is aligned nicely.